arenadata/Dockerfile.ubuntu

```diff
@@ -0,0 +1,60 @@
+FROM ubuntu:22.04 AS base
+
+COPY README.Ubuntu.bash ./
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN set -eux; \
+    ./README.Ubuntu.bash; \
+    rm README.Ubuntu.bash; \
+# The en_US.UTF-8 locale is needed to run GPDB
+    locale-gen en_US.UTF-8; \
+# To run sshd directly, but not using `/etc/init.d/ssh start`
+    mkdir /run/sshd; \
+# Alter precedence in favor of IPv4 during resolving
+    echo 'precedence ::ffff:0:0/96  100' >> /etc/gai.conf; \
+# Packages for tests
+    apt install -y rsync iproute2 krb5-kdc krb5-admin-server fakeroot lsof sudo python3.11 python3.11-dev python3-pip openjdk-11-jdk libipc-run-perl;\
+    pip install --upgrade pip; \
+# Install allure-behave for behave tests
+    pip3 install pytest gsutil behave~=1.2.6 coverage~=4.5 'mock<=5.0.0' allure-behave==2.4.0 psutil; \
```

Comment:
psutil is already installed by the above README.Ubuntu.bash script

Reply:
README.Ubuntu.bash installs python3-psutil which is for default python 3.10. Here we need 3.11 for tests, so install another one via pip

Reply:
Do we really need TWO pythons in the image?

Reply:
Idea was to not remove default one from README.Ubuntu.bash. Do you suggest to replace it?

Reply:
Will gpdb work only with those packages that are installed by the README.Ubuntu.bash script?

Reply:
Yes. Here are the packages required for the full testing.

Reply:
> README.Ubuntu.bash installs python3-psutil which is for default python 3.10. Here we need 3.11 for tests, so install another one via pip

```sh
pip3 --version
pip 22.0.2 from /usr/lib/python3/dist-packages/pip (python 3.10)
```